source/MRVoxels/MRMinimalSurface.cpp

```diff
@@ -83,6 +84,128 @@ DimsAndSize getDimsAndSize( const Vector3f& size, float frequency, float resolut
     return { dims, voxelSize };
 }
 
+
+std::map<float, float> reverseMap( const std::map<float, float>& other )
+{
+    std::map<float, float> res;
+    for ( auto [k, v] : other )
+        res[v] = k;
+    return res;
+}
+
+const std::vector<std::map<float, float>> density2iso = {
```

Comment:
better put it in a function as a static variable, to initialize it not on program startup, but on first function call